api/dataservices/edgejob/edgejob.go

```diff
@@ -94,3 +94,13 @@ func (service *Service) DeleteEdgeJob(ID portainer.EdgeJobID) error {
 func (service *Service) GetNextIdentifier() int {
 	return service.connection.GetNextIdentifier(BucketName)
 }
+
+// GetIdentifier returns the current max identifier for the datastore
+func (service *Service) GetIdentifier() int {
+	return service.connection.GetIdentifier(BucketName)
+}
+
+// SetIdentifier sets the current max identifier for the datastore
```

Comment:
This comment is not actually correct. It sets what we pass to the method not the max